src/components/timeline/XAxis.svelte

```diff
@@ -3,24 +3,47 @@
 <script lang="ts">
   import type { ScaleTime } from 'd3-scale';
   import ConstraintViolations from './ConstraintViolations.svelte';
+  import RowXAxisTicks from './RowXAxisTicks.svelte';
   import XAxisVerticalGuides from './XAxisVerticalGuides.svelte';
 
   export let constraintViolations: ConstraintViolation[] = [];
-  export let drawHeight: number = 90;
+  export let drawHeight: number = 70;
   export let drawWidth: number = 0;
   export let marginLeft: number = 50;
   export let verticalGuides: VerticalGuide[] = [];
   export let viewTimeRange: TimeRange | null = null;
   export let xScaleView: ScaleTime<number, number> | null = null;
   export let xTicksView: XAxisTick[] = [];
 
-  let axisOffset = 24;
-  let violationsOffset = 16;
-  let verticalGuidesOffset = 10;
+  let axisOffset = 12;
+  let violationsOffset = 0;
+  let verticalGuidesOffset = 0;
```

Comment:
So Jeff is working on this but for this commit can we put the current vertical guides on the same row as the cursor for now?
<img width="181" alt="Screen Shot 2022-10-17 at 12 25 17 PM" src="https://user-images.githubusercontent.com/683355/196265059-7b67c63c-6f0b-47ee-9f75-bbedd9627f46.png">

Reply:
Ah yes, I'll fix that